pkgs/top-level/all-packages.nix

```diff
@@ -5356,7 +5356,8 @@ in
 
   grocy = callPackage ../servers/grocy { };
 
-  nextcloud = callPackage ../servers/nextcloud { };
```

Comment:
Should we do a `throw` for migration also?

Reply:
In that case, `pkgs ? nextcloud` would be true, right? (for further context about this, I recommend to read https://github.com/NixOS/nixpkgs/pull/82353#issuecomment-598269471 since @fpletz and I already had a fairly long discussion about possible corner-cases :D)

Reply:
Awesome